admin_guide/limits.adoc

```diff
@@ -84,15 +83,71 @@ specified.
 <12> The default amount of CPU that a container will request to use if not specified.
 <13> The default amount of memory that a container will request to use if not specified.
 <14> The maximum amount of CPU burst that a container can make as a ratio of its limit over request.
-====
+endif::[]
+
+ifdef::openshift-online[]
+.Core Limit Range Object Definition
+
+[source,yaml]
+----
+apiVersion: "v1"
+kind: "LimitRange"
+metadata:
+  name: "core-resource-limits" <1>
+spec:
+  limits:
+    - type: "Pod"
+      max:
+        cpu: "2" <2>
+        memory: "3Gi" <3>
+      min:
+        cpu: "200m" <4>
+        memory: "6Mi" <5>
+    - type: "Container"
+      max:
+        cpu: "2" <6>
+        memory: "1Gi" <7>
+      min:
+        cpu: "100m" <8>
+        memory: "4Mi" <9>
+      default:
+        cpu: "300m" <10>
+        memory: "200Mi" <11>
+      defaultRequest:
+        cpu: "200m" <12>
+        memory: "100Mi" <13>
+      maxLimitRequestRatio:
+        cpu: "10" <14>
+----
+<1> The name of the limit range object.
+<2> The maximum amount of CPU that a pod can request on a node across all
+containers.
+<3> The maximum amount of memory that a pod can request on a node across all
+containers. For {product-title} Starter, the maximum is `2Gi` and for
+{product-title} Pro the maximum is `3Gi`.
+<4> The minimum amount of CPU that a pod can request on a node across all
+containers.
+<5> The minimum amount of memory that a pod can request on a node across all
+containers.
+<6> The maximum amount of CPU that a single container in a pod can request.
+<7> The maximum amount of memory that a single container in a pod can request.
+<8> The minimum amount of CPU that a single container in a pod can request.
+<9> The minimum amount of memory that a single container in a pod can request.
+<10> The default amount of CPU that a container will be limited to use if not
+specified.
+<11> The default amount of memory that a container will be limited to use if not specified.
+<12> The default amount of CPU that a container will request to use if not specified.
+<13> The default amount of memory that a container will request to use if not specified.
+<14> The maximum amount of CPU burst that a container can make as a ratio of its limit over request.
+endif::[]
```

Comment:
Due to the current limitrange.yaml is not an Online limitrange, I can not make sure whether the items need to be updated.